scripts/godice.mjs

```diff
@@ -24,22 +23,24 @@ Hooks.once('init', () => {
         restricted: false
     });
 
+    // Core Dice Configuration
+    CONFIG.Dice.fulfillment.methods.godice = { label: "GoDice Bluetooth Dice", interactive: true };
```

Comment:
If you have an icon that looks good at about 16x16, you can include an `icon` property here with the URL to it, and it will be used in the spawned `RollResolver` app.